oemoflex/postprocessing.py

```diff
@@ -13,6 +13,8 @@
 from oemoflex.helpers import delete_empty_subdirs, load_elements
 from oemoflex.preprocessing import get_parameter_values
 
+from oemoflex.facades import TYPEMAP
```

Comment:
At my system, it also works without this line (see https://github.com/modex-flexmex/oemo-flex/pull/48#issuecomment-646692379)

Reply:
The updated TYPEMAP is known anyway. Can you check this again on yours, @jnnr ?

Reply:
It works because when importing TYPEMAP from oemoflex.scalars, TYPEMAP gets updated, which affects facades.TYPEMAP as well. Flake8 complain about this import (line 16) not being used anywhere, even if this line is necessary. To make it more clear, I suggest to drop the 'from oemof.tabular import facades' and use the TYPEMAP only.